pkg/controller/resourcequota/resource_quota_controller_test.go

```diff
@@ -103,11 +103,11 @@ func TestFilterQuotaPods(t *testing.T) {
 			Status: api.PodStatus{Phase: api.PodFailed},
 		},
 	}
-	expectedResults := util.NewStringSet("pod-running",
+	expectedResults := sets.NewString("pod-running",
```

Comment:
It looks like this line caused the CI failure for `verify-flags-underscore.py`. Should I add it to `known-flags.txt` like the error message says?